include/RAJA/policy/cuda/MemUtils_CUDA.hpp

```diff
@@ -49,6 +49,25 @@ namespace RAJA
 namespace cuda
 {
 
+//! Get the properties of the current device
+RAJA_INLINE
+cudaDeviceProp get_device_prop()
```

Comment:
Should this be in the detail namespace?

Reply:
Perhaps, but this was just moved up in the file without changing the namespace that it is in so I'm inclined to leave it where it is and we can do a cleanup pass later if we think its worth changing.